integration-tests/src/test/java/tachyon/client/FileOutStreamIntegrationTest.java

```diff
@@ -219,6 +226,22 @@ private void writeTest3Util(TachyonURI filePath, ClientOptions op, int len) thro
   }
 
   /**
+   * Test writing to a file and specify the location to be localhost.
+   *
+   * @throws IOException if file can not be opened successfully.
+   */
+  @Test
+  public void writeSpecifyLocalTest() throws IOException, TException {
+    TachyonURI filePath = new TachyonURI(PathUtils.uniqPath());
+    final int LENGTH = 2;
+    FileOutStream os = mTfs.getOutStream(filePath, sWriteLocal);
+    os.write((byte) 0);
+    os.write((byte) 1);
+    os.close();
```

Comment:
It would be good to update the functionality of `TachyonFSTestUtils` to use options in a separate PR. (Created Ticket 1040)